dataeng/resources/warehouse-transforms.sh

```diff
@@ -61,3 +61,47 @@ then
     # Run all tests which haven't been excluded.
     dbt test --models tag:$MODEL_TAG $exclude_param --profile $DBT_PROFILE --target $DBT_TARGET --profiles-dir $WORKSPACE/analytics-secure/warehouse-transforms/
 fi
+
+# Each transfer is two elements in this array:
+#     - the DBT macro name
+#     - the macro arguments to use when calling the DBT macro
+PROGRAMS_REPORTING_TRANSFERS=( \
```

Comment:
I'm wondering if instead of putting this data here and having people edit config in yet another repo (4 all together for this PR I believe) could we put this configuration in to the DBT layer? If we could figure something out there it would allow us to remove at least 1 repos worth of PRs while creating a new s3 transfer.

Reply:
That would also enable us to DRY this code up some more. If we did so we could have a single jenkins job that runs all the s3 exports and pushes each functional area or project to an sub job to be executed. 

Also I'm wondering about execution schedule... Should this be in here at all? Given that different teams and functions may want to execute these exports on different schedules and different times we might be better served by moving this functionality to prefect?

Reply:
Re: editing config here:
Agreed that this is not ideal. We should minimize the number of places to edit config as much as possible. That means we'd move this config to warehouse-transfers somehow (maybe a seed? a macro included in other macros?) so that it could be editable by other teams (as opposed to analytics-secure).

Re: a single Jenkins job & execution schedule:
I separated the transfers into two batches because the transferred models were generated by two different Jenkins jobs - w-t-daily and w-t-enterprise (which is triggered after w-t-daily). I chose to keep the transfers happening after the models being transferred were built. My assumption was that newly-built models should be transferred immediately after being built. I guess we could have a separate "w-t-xfer-models" Jenkins job that gets triggered by multiple upstream jobs if we wanted?